numpy/lib/tests/test_arraysetops.py

```diff
@@ -286,6 +286,8 @@ def test_setdiff1d(self):
         assert_array_equal(c, ec)
 
         assert_array_equal([], setdiff1d([], []))
+        a = np.array((), np.uint32)
+        assert setdiff1d(a, []).dtype is np.dtype('uint32')
```

Comment:
Also `assert_`, not `assert`. You will need to add it to the functions imported from `numpy.testing` at the top of the file.